test/clojure-mode-font-lock-test.el

```diff
@@ -101,7 +101,13 @@ POS."
 
 (ert-deftest clojure-mode-syntax-table/fontify-clojure-keyword ()
   :tags '(fontification syntax-table)
-  (should (eq (clojure-test-face-at 2 11 "{:something 20}") 'clojure-keyword-face)))
+  (should (equal (clojure-test-face-at 2 11 "{:something 20}") '(clojure-keyword-face))))
```

Comment:
Because the keyword rule now uses the `append` option (see https://github.com/clojure-emacs/clojure-mode/pull/326/files#diff-50632fdcf67b6d3c8505a6aa071a2bd0R459). So font-lock-mode adds it as a list (even if it turns out to be the only face).